eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -54,9 +54,16 @@ jobs:
           versionSpec: "3.6"
       - script: |
           pip install setuptools wheel
-          pip install doc-warden
-          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
-        displayName: "Verify Readmes"
+          pip install doc-warden==0.4.2
+          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml -t all
+        displayName: "Verify Readmes and changelogs"
+        condition: ne(variables['Build.Reason'],'Manual')
+      - script: |
+          pip install setuptools wheel
+          pip install doc-warden==0.4.2
+          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml -t all -s release
+        displayName: "Verify Readmes, changelogs and latest release notes"
+        condition: eq(variables['Build.Reason'],'Manual')
```

Comment:
I think we might need to have a better condition for this. @mitchdenny do we have a variable that indicates we want to do a release? If not we should consider adding that similar to our setdevverison variable.

Reply:
Once we have that condition we should consider using an inline script (or pwsh) block that adds the extra arguments instead of duplicating this full task step.

Reply:
There is currently no specific variable that says that we want to do a release (a pipeline run is a pipeline run). That said I think we should consider having a Prep set of tasks in each job which look at the pipeline run and things like how it was queued, the branch and other factors and emits a bunch of variables which can be used to more clearly define the "intent" of the pipeline run.

Not sure I'd want to block this PR on getting that done though. @chidozieononiwu by the way I've bumped the version of Doc-Warden to 0.5.0 and its already running on master, so not sure whether you need this change anymore or not.

Reply:
I was waiting for version increment to be merged before getting back to this. But now changelog verification is also being moved out of Doc Warden. @weshaggard should I still add this changes in the mean time?

Reply:
I'd suggest not taking this change as it might just end up causing us to do part of this work multiple times. I suggest we just close this PR and enable the changelog validation in the new manner, we still have to wait for the version increment scripts to properly update the changelogs before we turn on this validation anyway.